docs/software/commands.md

```diff
@@ -379,7 +379,7 @@ format.
 
 ### The following HTTP commands are exposed on test instances
 * **generateload** `generateload[?mode=
-    (create|pay|pretend|mixed_txs|soroban_upload|soroban_invoke_setup|soroban_invoke|upgrade_setup|create_upgrade)&accounts=N&offset=K&txs=M&txrate=R&spikesize=S&spikeinterval=I&maxfeerate=F&skiplowfeetxs=(0|1)&dextxpercent=D&dataentrieslow=A&dataentrieshigh=B&kilobyteslow=C&kilobyteshigh=T&txsizelow=U&txsizehigh=V&cpulow=W&cpuhigh=X&instances=Y&wasms=Z]`
+    (create|pay|pretend|mixed_txs|soroban_upload|soroban_invoke_setup|soroban_invoke|upgrade_setup|create_upgrade|blend_classic_soroban_setup|blend_classic_soroban)&accounts=N&offset=K&txs=M&txrate=R&spikesize=S&spikeinterval=I&maxfeerate=F&skiplowfeetxs=(0|1)&dextxpercent=D&dataentriesintervals=A&dataentriesweights=B&kilobytesintervals=C&minpercentsuccess=S&kilobytesweights=T&txsizeintervals=U&txsizeweights=V&cpuintervals=W&cpuweights=X&instances=Y&wasms=Z&wasmbytesintervals=G&wasmbytesweights=H&payweight=P&sorobanuploadweight=Q&sorobaninvokeweight=R]`
```

Comment:
should we make the naming of modes consistent? Currently, we have "mixed txs" (mix of classic payments and DEX transactions), as well as "blend classic and soroban". Maybe we can do something like "mixed classic" and "mixed classic soroban"?

Reply:
Good idea. I renamed these modes to `mixed_classic` and `mixed_classic_soroban` in 9464b4c52fd72d24347b9cba6b224f04cbdb8eae